src/index.html

```diff
@@ -1,13 +1,319 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link href="https://fonts.googleapis.com/css2?family=Manrope&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/reset.css">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page_body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo">
+              <img src="images/logo-large.png" alt="Bose Logo">
+            </a>
+
+            <div class="top-bar__icons">
+              <a href="#" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+
+          <div class="header__bottom">
+            <h1 class="header__title">
+              Your senses.
+              <br>
+              Set free.
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="menu__content">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <img src="images/logo-large.png" alt="Bose Logo">
+          </a>
+
+          <a href="#" class="icon icon--close"></a>
+        </div>
+
+        <div class="menu__bottom">
+
+          <nav class="nav menu-nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#recommended">Recommended</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#categories">Categories</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">About Us</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">Contact Us</a>
+              </li>
+            </ul>
+          </nav>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+            +1 234 5555-55-55
+          </a>
+
+          <a href="tel:+1 234 5555-55-55" class="menu__call-to-order">
+            Call to order
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="main__content">
+        <div class="container">
+          <section id="recommended" class="recommended">
+            <h2 class="section-title">Recommended</h2>
+            <div class="recommended__products">
+              <article class="product recommended__product">
+                <img
```

Comment:
7. [CHECKLIST] Increase all images on hover (make them links to #, if there is no better option)

I see you added links to all other images except recommended section, please fix it